templates/Test.cpp.ejs

```diff
@@ -35,9 +35,16 @@ for (var i = 0; i < categorizedApi.otherCalls.length; i++)
     var call = categorizedApi.otherCalls[i];
     var testName = "Test" + api.name + call.name;
     let testStatusListStatus = testStatusMap[testName];
-    var skipTest = (testStatusListStatus != "Passing");
+    let passingWithDataTest = (testStatusListStatus == "PassingWithData");
```

Comment:
>== [](http://example.com/codeflow?start=52&length=2)

should all these be "==="?

Reply:
=== looks at datatype too, so not strictly required here, but wouldn't hurt.  i'll mess with this next pr